antiope-athena/src/Antiope/Athena.hs

```diff
@@ -41,8 +27,8 @@ query config context qstring clientRequestToken = do
     Nothing   -> return []
   where
     go qeid = do
-      accept <- await queryExecutionFinished (getQueryExecution qeid)
-      case accept of
+      accpt <- await queryExecutionFinished (getQueryExecution qeid)
+      case accpt of
```

Comment:
The extra `e` isn't that bad is it? 😁

Reply:
symbol shadowing complained by compiler :D

Reply:
Can we hide the offender instead of renaming?

Reply:
It's probably better to use `hiding` or qualified imports in this case.

Reply:
Fixed, thanks.